build.sh

```diff
@@ -0,0 +1,239 @@
+#! /bin/bash
+msBuild='/c/Windows/Microsoft.NET/Framework64/v4.0.30319/'
+outputFolder='./_output'
+outputFolderMono='./_output_mono'
+outputFolderOsx='./_output_osx'
+outputFolderOsxApp='./_output_osx_app'
+testPackageFolder='./_tests/'
+testSearchPattern='*.Test/bin/x86/Release/*'
+sourceFolder='./src'
+updateFolder=$outputFolder/NzbDrone.Update
+updateFolderMono=$outputFolderMono/NzbDrone.Update
+
+CheckExitCode()
+{
+    "$@"
+    local status=$?
+    if [ $status -ne 0 ]; then
+        echo "error with $1" >&2
+        exit 1
+    fi
+    return $status
+}
+
+CleanFolder()
+{
+    local path=$1
+    local keepConfigFiles=$2
+
+    echo "Removing XMLDoc files"
+    find $path -name "*.xml" -delete
+    find $path -name "*.transform" -delete
```

Comment:
The `-delete` flag on find isn't supported by all systems, most do, but I don't think we can expect it to work.